packages/client/src/search/transformer.ts

```diff
@@ -33,28 +51,31 @@ export const transformData = (
     return []
   }
 
-  return data.results
-    .filter((req): req is GQLEventSearchSet => req !== null)
-    .map((reg: GQLEventSearchSet) => {
-      let birthReg
-      let deathReg
-      let marriageReg
-      let names
-      let groomNames
-      let brideNames
-      let mergedMarriageName
-      let dateOfEvent
-      const assignedReg = reg as GQLEventSearchSet
-      if (assignedReg.registration && assignedReg.type === 'Birth') {
-        birthReg = reg as GQLBirthEventSearchSet
-        names = (birthReg && (birthReg.childName as GQLHumanName[])) || []
-        dateOfEvent = birthReg && birthReg.dateOfBirth
-      } else if (assignedReg.registration && assignedReg.type === 'Death') {
-        deathReg = reg as GQLDeathEventSearchSet
-        names = (deathReg && (deathReg.deceasedName as GQLHumanName[])) || []
+  const results = data.results.filter(
+    (req): req is GQLEventSearchSet => req !== null
+  )
+  return results.map((reg: GQLEventSearchSet) => {
```

Comment:
```suggestion
  return data.results
  .filter((req): req is GQLEventSearchSet => req !== null)
  .map((reg: GQLEventSearchSet) => {
```
Could we please change it to how it was previously, that way it would easier to reason about the changes in git diff otherwise it's quite difficult to follow :sweat_smile:

Reply:
Sure! Let's change it back😇